README.md

```diff
@@ -59,9 +54,6 @@ is:
       git:setup-remotes     Configures Git remote repositories.
      init
       init:manifest         Initializes a new phapp.yml for your project.
-     self
-      self:rollback         Rolls back to the previous version after a self-update.
```

Comment:
so let's keep supporting this or similar functionality based upon https://github.com/consolidation/self-update ?